server/routes/incidentDetails/incidentDetailsEdit.ts

```diff
@@ -46,10 +46,12 @@ export default class IncidentDetailsEditRoutes {
   private getAssociatedPrisonerNumber = (
     selectedRadio: string,
     previouslyReportedAssociatedPrisoner: string,
-    newAssociatedPrisoner: string
+    newAssociatedPrisoner: string,
+    personDeleted: string
   ): string => {
     if (selectedRadio === 'onTheirOwn' || selectedRadio === 'attemptOnTheirOwn') return null
-    if (newAssociatedPrisoner !== undefined) return newAssociatedPrisoner
+    if (personDeleted === 'true') return null
```

Comment:
Do we actually need this (prob do, but can't quite see why).. if the selectedPerson is always set then we just need to check whether that exists, don't we?